webrob/pages/main.py

```diff
@@ -64,7 +71,31 @@ def render_main():
     
     #return render_template('main.html', **locals())
     return redirect(url_for('render_QA_page'))
+    
+    
+# get call handling method for changing password
+@app.route('/change_password_get')
+@login_required
+def render_change_password_get():
+    form = PasswordForm()
+    return render_template('flask_user/change_password.html', title='Change Password', form=form)
 
+# post call handling method for changing password    
+@app.route('/change_password_post', methods=["POST"])
+@login_required
+def render_change_password_post():
+    form = PasswordForm()
+    if "Update" in request.form:
+        user = current_user
+        user.password = app.user_manager.hash_password(form.password.data)
+        db.session.add(user)
+        db.session.commit()
+        os.write(2, b'  Password has been updated! \n')
```

Comment:
any reason not to use flask logging interface `app.logger.info`?

Reply:
nope, I just copied it from one of the previous classes. :)

Reply:
I have changed the logger to more standard app.logger.info